vault/resource_database_secret_backend_connection_test.go

```diff
@@ -35,7 +35,7 @@ func TestAccDatabaseSecretBackendConnection_import(t *testing.T) {
 					resource.TestCheckResourceAttr("vault_database_secret_backend_connection.test", "postgresql.0.connection_url", connURL),
 					resource.TestCheckResourceAttr("vault_database_secret_backend_connection.test", "postgresql.0.max_open_connections", "2"),
 					resource.TestCheckResourceAttr("vault_database_secret_backend_connection.test", "postgresql.0.max_idle_connections", "0"),
-					resource.TestCheckResourceAttr("vault_database_secret_backend_connection.test", "postgresql.0.max_connection_lifetime", "0"),
+					resource.TestCheckResourceAttr("vault_database_secret_backend_connection.test", "postgresql.0.max_connection_lifetime", "0s"),
```

Comment:
I didn't run all the acceptance tests, but I did run this one and it failed for me. Here are the steps to reproduce:
- Run Vault locally using `$ vault server -dev`, and use the outputted information to do `$ export VAULT_TOKEN=whatever` and `$ export VAULT_ADDR=http://127.0.0.1:8200`
- The version of Vault I'm running is 0.11.1
- Run postgres locally, and do `$ export POSTGRES_URL=postgresql://{{username}}:{{password}}@localhost:5432/`

With that, I initially got this test failure that should be unrelated to your code and is merely due to redacting output:
```
--- FAIL: TestAccDatabaseSecretBackendConnection_import (0.04s)
    testing.go:434: Step 0 error: Check failed: Check 7/10 error: vault_database_secret_backend_connection.test: Attribute 'postgresql.0.connection_url' expected "postgresql://postgres:postgres@localhost:5432/", got "*****ql://*****:*****@localhost:5432/"
```
So to get around it, I just commented out line 35 of that file, and re-ran the test, and I got:
```
--- FAIL: TestAccDatabaseSecretBackendConnection_import (0.03s)
    testing.go:434: Step 0 error: Check failed: Check 9/9 error: vault_database_secret_backend_connection.test: Attribute 'postgresql.0.max_connection_lifetime' expected "0s", got "0"
```

That does look relevant and I bet it'd happen on all of the tests added. Could you take a look at that?

Reply:
Hey @tyrannosaurus-becks , just had a look at this.

Give me a few days to take this up again and make it merge-ready.